src/form/acp.jl

```diff
@@ -1,6 +1,25 @@
 # Three-phase specific constraints
 
 
+""
+function variable_tp_voltage(pm::PMs.GenericPowerModel{T}; nw=pm.cnw, kwargs...) where T <: PMs.AbstractACPForm
+    for c in PMs.conductor_ids(pm)
+        PMs.variable_voltage(pm, cnd=c; nw=nw, kwargs...)
+    end
+    # This is needed for delta loads, where division occurs by the difference
+    # of voltage phasors. If the voltage phasors at one bus are initialized
+    # in the same point, this would lead to division by zero.
```

Comment:
Another solution to this that would use the PowerModels infrastructure would be to check if the field `va_start` exists in the data model.  If not, add the default starting vector that you propose.  

We can also considering making these standard angle offsets a standard across the data model.  Open an issue to look into that.